FinalProject/LabFinalProject.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Final Project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The goal of the final project is to analyze functional imaging data from start to finish using the skills you learned in class. Below are the sections we would like you to complete as you work through the analyses.\n",
+    "\n",
+    "The project is worth 20 points based on the following components:\n",
+    "* Rationale and discussion\n",
+    "* Description of procedures, imaging acquisition, and quality control\n",
+    "* Preprocessing \n",
+    "* First-level analyses\n",
+    "* Group analyses and inference\n",
+    "\n",
+    "Each component is worth 5 points (5=all complete and accurate, 4=several minor\n",
```

Comment:
just realized i goofed and this doesn't add up. i'd proposed we keep the 5 separate parts above but then have their component points as:
Each component is worth 4 points (4=all complete and accurate, 3=several minor errors, 2=several major errors and/or many minor errors, 1=many major errors or incomplete).